docs/changelog/115675.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 115675
+summary: Removing `ChunkingSettingsFeatureFlag`
```

Comment:
This might end up in a release notes at some point, so lets replace this with a description of what removing the feature flag will do.

Reply:
Discussed offline and this can be switched to a non-issue since users wouldn't have seen the feature flag so there's no reason to have a changelog for this. I'll make that update.